rpm/SPECS/openresty.spec

```diff
@@ -218,6 +228,10 @@ fi
 /usr/bin/resty
 %{orprefix}/bin/resty
 
+%files devel
+%defattr(-,root,root,-)
+
+/usr/include/nginx/*.h
```

Comment:
@madisodr The installation location for the header files should be `%{or_prefix}/nginx/include`. Yes, we also need those automatically generated header files in nginx's `objs/` directory.